### PR TITLE
Port fixes from rambox-whatsapp-shortcuts-telegram-theme

### LINKED ISSUE #1: Does not work with recent WhatsApp
This plugin does not work with recent versions of WhatsApp.  The fork here does.  I am working on porting the fixes from there.

### PR DESCRIPTION
**Pull request by andybalaam**
With these changes this works for me in latest WhatsApp.  This fixes #1

### CODE REVIEW

File: whatsappium.js [hunk 1]
```diff
@@ -8,32 +8,31 @@ function dispatchClick (item) {
...
     dispatchClick(chat);
   }
 }
 
 
 function isItemActive (item) {
-    const chat = item.querySelector('.chat');
-    return chat && chat.classList.contains('active');
+    const chat = item.querySelector('._2EXPL');
+    return chat && chat.classList.contains('_1f1zm');
```
Comment (MichaelAquilina) [inline whatsappium.js:20]: hmmm it seems like these are derived CSS classes (maybe they are using something like SASS). Is there anything else we can use that is a bit more human readable and less likely to change over time?

Comment (andybalaam) [reply whatsappium.js:20]: Yes, it is annoying, and will likely change if they ship a rebuilt version.  Before I found this work I had a quick go at doing it myself and there is nothing obvious to identify the right elements.  We could do it with expressions like "div[dir=auto] > div > etc.... > div" but they would likely be fragile too.

Comment (andybalaam) [reply whatsappium.js:20]: For reference, this change was made on 6th October 2018 https://github.com/c-samuel/rambox-whatsapp-shortcuts-telegram-theme/commit/4d8394e558c41b1b59e77e108f135849d1d150db and it still works ... for now.

**Review by MichaelAquilina — APPROVED** [review]
Happy to merge this for now and find a longer term solution later.

Will need to upload the new file later on in the evening.

Thanks for the PR!

**Comment by andybalaam** [comment]
Awesome, thanks!

**Pull request merged**